travharv/helper.py

```diff
@@ -18,7 +18,8 @@ def makeNSM(pfx_declarations: Dict[str, str]) -> Dict[str, Namespace]:
 
 
 def resolve_uri(uri: str, nsm: NamespaceManager) -> URIRef:
-    # TODO look into python library that can resolve URN's (urnparse was not good)
+    # TODO look into python library that can resolve URN's
+    # (urnparse was not good)
     if uri.startswith("urn:"):
         uri = uri.replace("urn:", "http://make.safe/")
```

Comment:
Since I presume that you will implement urnparse sulotion in py-rdf-store I will not include these changes in travharv and instead like you suggest let py-rdf-store.clean handle this functionality